src/tigerbeetle/cli.zig

```diff
@@ -856,7 +866,11 @@ fn parse_cache_size_to_count(
         value_count_max_multiple,
     ) * value_count_max_multiple;
 
-    const result: u32 = @intCast(count_rounded); // TODO: better error message on overflow
```

Comment:
> This will probably require adding the name of the CLI flag as a parameter to produce a nice error message.

In my limited knowledge of the code base, won't extending `parse_cache_size_to_count` to include the name of the CLI flag feels a bit tacky? But I am fine adding it, if it fits with the style of the codebase.

The other way could be to do some comptime mapping of first argument to the CLI flag. ie. `tigerbeetle.Account => cache-accounts`, `tigerbeetle.Transfer => cache-transfers` etc.

Let me know which approach is the more preferred :)

Reply:
Adding a string argument is the way to go, as that's the simplest solution. 

Returning an error via `!` is problematic, because it gives the caller a choice between propagating the error further up the stack or handling it, but, in this case, we don't need this choice, because we already know the right answer --- exit with fatal error.